integration_test/gce/gce_testing.go

```diff
@@ -739,6 +739,95 @@ func runCommand(ctx context.Context, logger *log.Logger, stdin io.Reader, args [
 	return output, err
 }
 
+// getGcloudConfigDir returns the current gcloud configuration directory.
+func getGcloudConfigDir(ctx context.Context) (string, error) {
+	null, err := os.Open(os.DevNull)
+	if err != nil {
+		panic(fmt.Sprintf("unable to open os.DevNull: %v", err))
+	}
+	out, err := RunGcloud(ctx, log.New(null, "", 0), "", []string{"info", "--format=value[terminator=''](config.paths.global_config_dir)"})
+	if err != nil {
+		return "", fmt.Errorf("error running gcloud info: %w", err)
```

Comment:
nit: RunGcloud's returned error will already include the full arguments list, no need to reiterate what it tried to do in the error message.

Reply:
Good point. Done.